meerk40t/core/elements/elements.py

```diff
@@ -1345,6 +1360,110 @@ def create_image(oplist):
         if needs_signal:
             self.signal("default_operations")
 
+    def create_usable_copy(self, sourceop):
+        from copy import copy
```

Comment:
Copy can go in the top, it's standard for python

Reply:
Done